lib/mayaUsd/nodes/proxyShapeBase.cpp

```diff
@@ -746,6 +747,37 @@ MayaUsdProxyShapeBase::isStageValid() const
     return true;
 }
 
+MStatus
+MayaUsdProxyShapeBase::preEvaluation(const MDGContext& context, const MEvaluationNode& evaluationNode)
+{
+    if (evaluationNode.dirtyPlugExists(excludePrimPathsAttr)) {
+        _IncreaseExcludePrimPathsVersion();
+    }
+    else if (evaluationNode.dirtyPlugExists(outStageDataAttr) ||
+        // All the plugs that affect outStageDataAttr
+        evaluationNode.dirtyPlugExists(filePathAttr) ||
```

Comment:
I understand that all of these attributes can be connected/animated?

Reply:
inStageDataCachedAttr being not writeable is enough that it can't be animated and so could be removed from this list.